bitcoin/rpc.py

```diff
@@ -509,13 +509,21 @@ def getblockhash(self, height):
                     (self.__class__.__name__, ex.error['message'], ex.error['code']))
 
     def getinfo(self):
+        try:
+           """Return a JSON object containing various state info"""
+           r = self._call('getinfo')
+           if 'balance' in r:
+              r['balance'] = int(r['balance'] * COIN)
+           if 'paytxfee' in r:
+              r['paytxfee'] = int(r['paytxfee'] * COIN)
+           return r
+        except:
+           print("getnetworkinfo replaces getinfo on versions > 0.16.0, please use getnetworkinfo()")
```

Comment:
This should print to stderr or maybe call depreciationwarning or similar.

Reply:
added DepreciationWarning seems to work well. Hope that works, otherwise let me know and I can add more. I'm very new to python so excuse my ignorance on some things.